File: src/app/admin/XXXXX.Admin.UI/src/domain/models/permissions/permissionCategory.ts
export class PermissionCategory {
    label: string;
    prefix: string;

    constructor(params: PermissionCategoryDTO) {
        this.label = params.label;
        this.prefix = params.prefix;
    }
}

export interface PermissionCategoryDTO {
    label: string;
    prefix: string;
}

File: src/shared/XXXXX.Domain/Models/Permissions/PermissionCategory.cs
using System;
using System.Collections.Generic;

namespace XXXXX.Domain.Models
{
    public class PermissionCategory: ITranslatable<TranslationPermissionCategory>
    {
        public string Label { get; set; }
        public string Prefix { get; set; }
        public List<TranslationPermissionCategory> Translations { get; set; }
    }

    public class TranslationPermissionCategory: ITranslation
    {
        public Guid LanguageId { get; set; }
        public string Label { get; set; }
    }
}